src/main/scala/rocket/tile.scala

```diff
@@ -30,7 +29,6 @@ class RocketTile(tileId: Int)(implicit p: Parameters) extends LazyModule {
   val dcacheParams = coreParams.alterPartial({
     case CacheName => CacheName("L1D")
     case TLId => "L1toL2"
-    case TileId => tileId // TODO using this messes with Heirarchical P&R: change to io.hartid?
```

Comment:
Excellent. Now we just need to covert the Icache and we won't need the TL1 L1toL2 config anymore in rocket at all.